src/test/resources/com/znsio/sample/e2e/features/calculator.feature

```diff
@@ -2,7 +2,7 @@
 Feature: Calculator test
 
 #  CONFIG=./configs/calculator_local_config.properties PLATFORM=android TAG=calculator ./gradlew run
-  @android
+  @android @web
```

Comment:
why web tag here? Make sure you do one round of review of Code and check what is being pushed.